Graphics Algos/README.md

```diff
@@ -0,0 +1,10 @@
+## GRAPHICS ALGORITHMS
+
+This is a folder which contains graphics algorithms and their implementations.
+
+### Robo.c
+
```

Comment:
upload your output pic in the Readme

Reply:
@prateekiiest added the output pic to the README.md file :)